acceptance/cloud_test.go

```diff
@@ -36,10 +38,12 @@ func (p *PluginTest) TestCloudLifecycle() {
 	t := p.T()
 
 	data := map[string]interface{}{
-		"auth_url":         "https://example.com/v3/",
-		"username":         tools.RandomString("us", 4),
-		"password":         tools.RandomString("", 15),
-		"user_domain_name": "Default",
+		"auth_url":          "https://example.com/v3/",
+		"username":          tools.RandomString("us", 4),
+		"password":          tools.RandomString("", 15),
+		"user_domain_name":  "Default",
+		"username_template": tools.RandomString("t", 5),
+		"password_policy":   tools.RandomString("p", 5),
```

Comment:
No need to generate template randomly, it's better to use some pre-selected value